client/src/components/Form.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import FormBasics from "./FormBasics";
+import FormAwards from "./FormAwards";
+import FormPublications from "./FormPublications";
+import FormLanguages from "./FormLanguages";
+import FormReferences from "./FormReferences";
+import FormWork from "./FormWork";
+import FormVolunteer from "./FormVolunteer";
+import FormEducation from "./FormEducation";
+import FormSkills from "./FormSkills";
+import FormInterests from "./FormInterests";
+
```

Comment:
That's a lot of components! Think about reusability when it comes to components - is there a way to pass props down to facilitate reusing some code? Maybe props.children as well.